app/Jobs/CreateRockTheVotePostInRogue.php

```diff
@@ -210,6 +210,23 @@ private function parseReferralCode($referralCode)
         return $finalValues;
     }
 
+    /**
+     * Parse a CSV field value as boolean.
+     *
+     * @param string $value
+     * @return boolean
+     */
+    private function parseBoolean($value) {
```

Comment:
@DoSomething/team-bleed Is there a better place to add this function, to potentially re-use in other imports?

Reply:
@aaronschachter we don't currently have a helper file because I think so far everything has been specific to a particular import, but this could be a good time to make one if you want to ☕️